Formula/cgit.rb

```diff
@@ -0,0 +1,36 @@
+class Cgit < Formula
+  desc "Hyperfast web frontend for Git repositories written in C"
+  homepage "https://git.zx2c4.com/cgit/"
+  url "https://git.zx2c4.com/cgit/snapshot/cgit-1.2.tar.xz"
+  sha256 "d5622ba7c3eb87de402da6fe3a092219546a917ff627966e99c9f18ba32a4832"
+
+  depends_on "gettext"
+  depends_on "openssl"
+
+  resource "git" do
+    url "https://www.kernel.org/pub/software/scm/git/git-2.18.0.tar.gz"
```

Comment:
Can you update to the latest version?
Other than that, it looks OK to merge.

Reply:
@fxcoudert The CGit requires fixed version of Git to work. See https://git.zx2c4.com/cgit/tree/Makefile?h=v1.2#n17